pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -52,24 +58,35 @@
 public class GrpcSendingMailbox implements SendingMailbox {
   private static final Logger LOGGER = LoggerFactory.getLogger(GrpcSendingMailbox.class);
 
+  private static final List<ByteString> EMPTY_BYTEBUFFER_LIST = Collections.emptyList();
+
+  private final PinotConfiguration _config;
```

Comment:
Do you need the attribute? it looks like it is only used in the constructor

Reply:
Yeah, didn't remove it after moving some code to the constructor. Thanks for removing it.